src/lib/Behat/BusinessContext/ObjectStatesContext.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzPlatformAdminUi\Behat\BusinessContext;
+
+use Behat\Gherkin\Node\TableNode;
+use EzSystems\EzPlatformAdminUi\Behat\PageElement\Dialog;
+use EzSystems\EzPlatformAdminUi\Behat\PageElement\ElementFactory;
+use EzSystems\EzPlatformAdminUi\Behat\PageObject\ObjectStateGroupPage;
+use EzSystems\EzPlatformAdminUi\Behat\PageObject\PageObjectFactory;
+use PHPUnit\Framework\Assert;
+use PHPUnit\Framework\AssertionFailedError;
+
+class ObjectStatesContext extends BusinessContext
+{
+    /**
+     * @Then there's :objectStateName on :objectStateGroupName Object States list
+     */
+    public function verfyObjectStateIsOnList(string $objectStateName, string $objectStateGroupName): void
+    {
+        $objectStateGroupPage = PageObjectFactory::createPage($this->utilityContext, ObjectStateGroupPage::PAGE_NAME, $objectStateGroupName);
+        $objectStateGroupPage->verifyIsLoaded();
+        $objectStateExists = $objectStateGroupPage->adminLists['Object States']->table->isElementInTable($objectStateName);
+
+        if (!$objectStateExists) {
+            Assert::fail(sprintf('Element "%s" is not on the "%s" object states list.', $objectStateName, $objectStateGroupName));
+        }
+    }
+
+    /**
+     * @Then there's no :objectStateName on :objectStateGroupName Object States list
+     */
+    public function verfyObjectStateIsNotOnList(string $objectStateName, string $objectStateGroupName): void
+    {
+        $objectStateGroupPage = PageObjectFactory::createPage($this->utilityContext, ObjectStateGroupPage::PAGE_NAME, $objectStateGroupName);
+        $objectStateGroupPage->verifyIsLoaded();
+        try {
+            $objectStateGroupPage->verifyListIsEmpty();
```

Comment:
How about adding a new method: `isListEmpty` (or getItemCount, it's up to you), that will be used by verifyListIsEmpty and here? Catching the Exception seems weird.